src/ServiceControlInstaller.Engine/Unattended/UnattendServiceControlSplitter.cs

```diff
@@ -81,7 +81,7 @@ Result ValidateUpgradeAction(ServiceControlInstance instance)
                 case RequiredUpgradeAction.Upgrade:
                     return Result.Failed("This instance cannot have an Audit instance split from it. Upgrade the instance instead", RequiredUpgradeAction.Upgrade);
                 case RequiredUpgradeAction.ConvertToAudit:
-                    return Result.Failed("This instance cannot have an Audit instance split from it. Convert the instance into an Audit instance instead", RequiredUpgradeAction.ConvertToAudit);
+                    return Result.Failed("This instance cannot have an Audit instance split from it. This instance has no recommended upgrade action", RequiredUpgradeAction.ConvertToAudit);
```

Comment:
@mikeminutillo would it be possible to add the discussed "read more on our docs site or contact support" kind of message?

Reply:
Done